test/app_test.dart

```diff
@@ -185,4 +187,97 @@ Future<void> main([List<String>? args]) async {
     final user1 = await app.logIn(Credentials.emailPassword(testUsername, testPassword));
     expect(app.users, [user1, user]);
   });
+
+  baasTest('AppConfiguration.logger', (configuration) async {
+    final logger = Logger.detached(generateRandomString(10))..level = LogLevel.all.loggerLevel;
+    configuration = AppConfiguration(
+      configuration.appId,
+      logLevel: LogLevel.all,
+      logger: logger,
```

Comment:
I think it's kind of annoying we have both of these. Can we infer the sync loglevel from the logger level and use that?

Reply:
Done